src/traffic_cache_tool/Makefile.inc

```diff
@@ -31,22 +31,11 @@ traffic_cache_tool_traffic_cache_tool_SOURCES = \
     traffic_cache_tool/CacheTool.cc \
     traffic_cache_tool/File.h \
     traffic_cache_tool/File.cc \
-    traffic_cache_tool/Command.h \
-    traffic_cache_tool/Command.cc \
     traffic_cache_tool/CacheScan.h \
     traffic_cache_tool/CacheScan.cc
 
 traffic_cache_tool_traffic_cache_tool_LDADD = \
-    $(top_builddir)/src/tscore/.libs/ink_assert.o \
-    $(top_builddir)/src/tscore/.libs/ink_code.o \
-    $(top_builddir)/src/tscore/.libs/ink_error.o \
-    $(top_builddir)/src/tscore/.libs/ink_file.o \
-    $(top_builddir)/src/tscore/.libs/ink_memory.o \
-    $(top_builddir)/src/tscore/.libs/ink_string.o \
-    $(top_builddir)/src/tscore/.libs/BufferWriterFormat.o \
+    $(top_builddir)/src/tscore/libtscore.la \
```

Comment:
I'm concerned about this. One of the goals is to have a cache tool that can run without traffic server or with different versions of traffic server, so that it is more portable and reliable. Linking with a dynamic `tscore` defeats that. Another option is to look at providing a static `tscore` and linking with that. I haven't had the time to investigate doing that.

Reply:
I see.